src/main/java/teammates/ui/controller/InstructorCourseInstructorEditSaveAction.java

```diff
@@ -216,24 +223,39 @@ private void removeSessionLevelPrivileges(InstructorAttributes instructorToEdit,
         return sectionNamesMap;
     }
 
-    private void updateInstructorPrivilegesForSectionInSectionLevel(String sectionParam, List<String> sectionNames, InstructorAttributes instructorToEdit) {
-        boolean isViewStudentInSectionsChecked = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_STUDENT_IN_SECTIONS + sectionParam) != null;
-        boolean isViewCommentInSectionsChecked = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_PERMISSION_VIEW_COMMENT_IN_SECTIONS + sectionParam) != null;
-        boolean isGiveCommentInSectionsChecked = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_PERMISSION_GIVE_COMMENT_IN_SECTIONS + sectionParam) != null;
-        boolean isModifyCommentInSectionsChecked = getRequestParamValue(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COMMENT_IN_SECTIONS + sectionParam) != null;
+    private void updateInstructorPrivilegesForSectionInSectionLevel(
+            String sectionParam, List<String> sectionNames, InstructorAttributes instructorToEdit) {
```

Comment:
I'm following the section before that, the one saying when all else fails just indent 8 spaces.

P.S. and also because I'm lazy, 8 spaces is automatically there when I press enter :p